features/bootstrap/FeatureContext.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+use Behat\Behat\Tester\Exception\PendingException;
+use Behat\Behat\Context\Context;
+use Behat\Gherkin\Node\PyStringNode;
+use Behat\Gherkin\Node\TableNode;
+
+/**
+ * Defines application features from the specific context.
+ */
+class FeatureContext extends CommandFeature implements Context
+{
+  private $importsPath = '';
+  private $exportsPath = '';
+
+  private $filename = '';
+
+
+  private $importResult = null;
+  private $exportResult = null;
+
+  /**
+   * Initializes context.
+   *
+   * Every scenario gets its own context instance.
+   * You can also pass arbitrary arguments to the
+   * context constructor through behat.yml.
+   */
+  public function __construct()
+  {
+    parent::__construct();
+
+    $this->importsPath = dirname(__FILE__).'/test_imports/';
+    $this->exportsPath = dirname(__FILE__).'/test_exports/';
+  }
+
+  private function testExitCode($result, $expected = 0) {
+    PHPUnit_Framework_Assert::assertEquals($expected, (int) $result['exitCode'], "Exit code does not match expected");
+  }
+
+  /**
+   * @Given a WP install
+   */
+  public function aWpInstall()
+  {
+    $result = $this->run("core is-installed");
+    $this->testExitCode($result);
+  }
+
+  /**
+   * @When I run the wp-cli command to import the file :arg1
+   */
```

Comment:
I would rather like see the whole command that was run instead of a description.